src/CMakeLists.txt

```diff
@@ -1091,7 +1091,6 @@ install(PROGRAMS
   ${CMAKE_RUNTIME_OUTPUT_DIRECTORY}/ceph-post-file
   ${CMAKE_SOURCE_DIR}/src/ceph-run
   ${CMAKE_SOURCE_DIR}/src/ceph-rest-api
-  ${CMAKE_SOURCE_DIR}/src/vstart.sh
```

Comment:
it's a developer only script. and only cmake installs it. so no. even if some of us are using it, i'd recommend him/her to run it from the source tree.